mlos_bench/mlos_bench/config/optimizers/one_shot_opt.jsonc

```
@@ -20,12 +22,12 @@
             // In which case, unspecified values are governed by the "use_default" config parameter.
             "tunables/tunable-values-example.jsonc"
         ],
-        // Additional values of the tunable parameters.
-        "tunables_values": {},
+        // Additional values of the tunable parameters, encoded as a list of dictionaries.
```

```suggestion
        // Additional values of the tunable parameters.
```

Future work to make that support multiples (for parity with `include_tunable_values: []`